tests/test_errors.py

```diff
@@ -57,7 +57,7 @@ class Model:
 
         clf = KerasClassifier(model=Model())
         with pytest.raises(TypeError, match="`model` must be"):
-            clf.fit(np.array([[0]]), np.array([0]))
+            clf.fit(np.array([[0], [1]]), np.array([0, 1]))
```

Comment:
All of these changes are just because the we were giving a new error since `clf.fit(np.array([[0]]), np.array([0]))` gave an unknown target type that was masking the original error.